universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -173,9 +176,19 @@ private ContainerTag debugContent() {
 
   private Tag loginButton(Messages messages) {
     String msg = messages.at(MessageKey.BUTTON_LOGIN.getKeyName());
-    return redirectButton(
-            "idcs", msg, routes.LoginController.idcsLoginWithRedirect(Optional.empty()).url())
-        .withClasses(BaseStyles.LOGIN_REDIRECT_BUTTON);
+    if (applicantIdp.equals(AuthIdentityProviderName.LOGIN_RADIUS_APPLICANT.getString())) {
+      return redirectButton(
```

Comment:
is there were we decide which applicant provider to use? with the different redirect calls? if so i think maybe it makes more sense to put this logic in the controller

Reply:
I think the *right* way to do it is to bind the correct provider in `SecurityModule.java` but for now we can make a generic controller `loginWithRedirect` method.

Reply:
Okay, moved this conditional logic to the controller and it looks a lot better!